tools/rc/ResourceCompiler.cpp

```diff
@@ -411,36 +413,54 @@ int main(int argc, char** argv)
     }
     return_code = EXIT_FAILURE;
   }
-  
-  option::Option* bundleFileOpt = options[BUNDLEFILE];
-  if (bundleFileOpt && bundleFileOpt->count() > 1 )
+
+  // multiple specifications of the following args are illegal.
+  auto check_multiple_args = [&options, &return_code](OptionIndex optionidx, const std::string& arg)
```

Comment:
I believe this lambda can be further simplified. ```const std::string& arg``` isn't redundant since the same text can be obtained from the ```option::Option``` object passed into the capture list (i.e. ```&options```). 
You could even take this a step further and pass in a vector of ```OptionIndex``` values instead of invoking the lambda ```N``` times.

With all these checks for invalid options combinations, it may make sense to re-factor this code to consolidate all this logic in one place.